test/circleci_tests.txt

```diff
@@ -1,16 +1,20 @@
-os_stds_methods/test_add_hvac_systems.rb
-os_stds_methods/test_service_water_heating.rb
-os_stds_methods/test_daylighting_controls.rb
-os_stds_methods/test_space_type.rb
-os_stds_methods/test_misc.rb
+90_1_general/test_179d.rb
 
-90_1_prm/test_prm_baseline_bldg.rb
-90_1_prm/test_prm_baseline_bldg4.rb
-90_1_prm/test_prm_baseline_bldg10.rb
-90_1_prm/test_prm_baseline_bldg11.rb
-90_1_prm/test_prm_baseline_bldg12.rb
-90_1_prm/test_prm_baseline_bldg13.rb
-90_1_prm/test_appendix_g_prm.rb
+90_1_prm/test_prm_baseline_bldg_179d.rb
+
+# os_stds_methods/test_add_hvac_systems.rb
+# os_stds_methods/test_service_water_heating.rb
+# os_stds_methods/test_daylighting_controls.rb
```

Comment:
Register our two tests and disable pretty much all of the others.

```
bundle exec ruby test/90_1_prm/test_prm_baseline_bldg_179d.rb
```

takes 6seconds